src/sc_crawler/vendors/alicloud.py

```
@@ -144,6 +146,73 @@ def _get_sku_prices(
+                **extra_request_params,
+            )
+            runtime = RuntimeOptions()
+            response = client.describe_available_resource_with_options(request, runtime)
+            if response.body:
+                available_zones = response.body.available_zones
+                if available_zones and available_zones.available_zone:
+                    resources = [
+                        resource.to_map()
+                        for resource in response.body.available_zones.available_zone
+                    ]
+            return region_id, resources
```

Ruff (0.14.13)

Consider moving this statement to an `else` block

(TRY300)